ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.hbs

```diff
@@ -29,4 +29,8 @@
     </div>
 </div>
 
+{{#is type 'custom'}}
```

Comment:
Instead of adding logic to the handlebars template, we should just have the javascript add a class called `is-custom` when the type is custom.  Then we can hide and show this in the less.

Reply:
I followed the precedent here: https://github.com/codice/ddf/blob/7c03479acc8b86f4d58bb45ae30011517cf8532b/ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.hbs#L15. Updated this code.